app/views/start-page.html

```diff
@@ -35,14 +35,13 @@ <h1 class="start-page-banner__text start-page-banner__heading govuk-heading-xl">
           <p class="start-page-banner__text start-page-banner__description govuk-body govuk-!-font-size-27">Register trainees with the Department for Education and record the outcome of their training.</p>
           <div class="start-page-banner__actions">
             {{ govukButton({
-              text: "Get started",
-              href: "/home",
+              text: "Sign in",
+              href: "sign-in",
```

Comment:
```suggestion
              href: "/sign-in",
```

What you did is not _wrong_, but this is slightly safer.